lib/src/seq.rs

```diff
@@ -39,6 +39,20 @@ macro_rules! declare_seq_with_contents_constraints_impl {
                 }
             }
 
+            #[cfg_attr(feature="use_attributes", unsafe_hacspec)]
+            pub fn with_capacity(l: usize) -> Self {
```

Comment:
So this creates a `Seq` of size 0 doesn't it? Why do you need that in hacspec, since pushing elements to a Seq is forbidden ?

Reply:
hm, good point. I can't exactly recall why/where I used this ... I opened an issue to see if we actually need this.